<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {background-color: rgba(255, 255, 255, 0%);
        border-image: url('bg2.jpg');}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="openButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>72</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton,
QToolButton {
    selection-color: none;
	background-color:none;
  
    border-radius: 3px;
}



QPushButton:hover,
QToolButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgb(222,222,222,5%), stop:1 rgb(222,222,222,10%));
}

QPushButton:pressed,
QToolButton:pressed,
QPushButton:on,
QToolButton:on {
    qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgb(0,0,0,18%), stop:1 rgb(0,0,0,25%));
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>72</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton,
QToolButton {
    selection-color: none;
	background-color:none;
  
    border-radius: 3px;
}



QPushButton:hover,
QToolButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgb(222,222,222,5%), stop:1 rgb(222,222,222,10%));
}

QPushButton:pressed,
QToolButton:pressed,
QPushButton:on,
QToolButton:on {
    qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgb(0,0,0,18%), stop:1 rgb(0,0,0,25%));
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton,
QToolButton {
    selection-color: none;
	background-color:none;
  
    border-radius: 3px;
}



QPushButton:hover,
QToolButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgb(222,222,222,5%), stop:1 rgb(222,222,222,10%));
}

QPushButton:pressed,
QToolButton:pressed,
QPushButton:on,
QToolButton:on {
    qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgb(0,0,0,18%), stop:1 rgb(0,0,0,25%));
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>70</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton,
QToolButton {
    selection-color: none;
	background-color:none;
  
    border-radius: 3px;
}



QPushButton:hover,
QToolButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgb(222,222,222,5%), stop:1 rgb(222,222,222,10%));
}

QPushButton:pressed,
QToolButton:pressed,
QPushButton:on,
QToolButton:on {
    qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgb(0,0,0,18%), stop:1 rgb(0,0,0,25%));
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="runButton">
     <property name="geometry">
      <rect>
       <x>804</x>
       <y>80</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton,
QToolButton {
    selection-color: none;
	background-color:none;
  
    border-radius: 3px;
}



QPushButton:hover,
QToolButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgb(222,222,222,5%), stop:1 rgb(222,222,222,10%));
}

QPushButton:pressed,
QToolButton:pressed,
QPushButton:on,
QToolButton:on {
    qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 rgb(0,0,0,18%), stop:1 rgb(0,0,0,25%));
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="codeout">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
font-size : 15px;


}</string>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="codein">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>411</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
font-size : 15px;


}</string>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="console">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>831</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
font-size : 15px;


}</string>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
